app/assets/stylesheets/components/common/_news_signup_inpage.scss

```diff
@@ -0,0 +1,531 @@
+          left: 50%;
+          transform: translate(-50%, -50%);
+          width: 100%;
+          margin: 10px auto;
+        }
+        @include respond-to($mq-m) {
+          position: relative;
+          top: auto;
+          left: auto;
+          transform: none;
+          margin: auto;
+        }
+        @include respond-to($mq-3col-layout) {
+          position: absolute;
+          top: 50%;
+          left: 50%;
+          transform: translate(-50%, -50%);
+          width: 100%;
+        }
+        @include respond-to($mq-l) {
+          position: relative;
+          top: auto;
+          left: auto;
+          transform: none;
+          margin: auto;
+        }
+      }
+
+    }
+
+    .news-signup-inpage__supplementary-text {
+      @include body(16, 24);
+      color: $color-heading-extra-small;
+      text-align: center;
+      font-weight: 800;
+      margin-top: $baseline-unit*3;
+
+      @include respond-to($mq-s) {
+        margin-top: 0;
+      }
+
+      @include respond-to($mq-m) {
+        margin-bottom: 0;
+      }
+
+      @include respond-to($mq-l) {
+        @include body(18, 24);
+      }
+
+    }
+
+    .news-signup-inpage__email-label {
+      @include visually-hidden;
+
+    }
+
+    .news-signup-inpage__image {
+      display: block;
+      position: absolute;
+      background-color: transparent;
+      background-image: image_url('news_signup_inpage/paper_clip_image.png');
+      background-repeat: no-repeat;
+      background-size: cover;
+      right: 0;
+      width: 155px;
+      height: 151px;
+      margin-left: -10px;
+      margin-top: -5px;
+
+      @include device-pixel-ratio() {
+        background-image: image_url('news_signup_inpage/paper_clip_image-2x.png');
+      }
+
+      @include respond-to($mq-m) {
+        right: auto;
+        left: 0;
+      }
+
+    }
+
+  }
+
+  .news-signup-inpage__form-container {
+    @include clearfix;
+    display: block;
+    position: relative;
+    text-align: center;
+
+    @include respond-to($mq-m) {
+      margin-top: 30px;
+    }
+
+    @include respond-to($mq-3col-layout) {
+      margin-top: 20px;
+    }
+
+    @include respond-to($mq-xl) {
+      margin-top: 23px;
+    }
+
+    input[type="email"] {
+      position: relative;
+      top: 1px;
+      width: 70%;
+      height: 38px;
+      border-color: $color-white;
+      border-radius: 4px;
+      margin: 0 0 $baseline-unit*2 auto;
+      text-align: center;
+
+      @include respond-to($mq-s) {
+        width: 45%;
+        margin-right: $baseline-unit;
+        margin-bottom: 0;
+        text-align: left;
+      }
+
+      @include respond-to($mq-3col-layout) {
+        width: 75%;
+        margin-right: 0;
+        margin-bottom: $baseline-unit*2;
+        text-align: center;
+      }
+
+      @include respond-to($mq-l) {
+        width: 40%;
+        margin-right: $baseline-unit;
+        margin-bottom: 0;
+        text-align: left;
+      }
+    }
+
+    .button--newsletter {
```

Comment:
This is where the layout vs component stuff gets a bit tricky. Strictly speaking components shouldn't be styling other components. In this situation you can use a layout class (e.g. `l-news-signup-inpage`) which will position and style elements like this one does. We sort of accepted when coming up with this concept that we would sacrifice additional wrapping divs for clarity in the CSS. Might be easier to chat through this one when you have a minute.